### ISSUE TITLE
Progress bar not visible on smaller displays 

### ISSUE THREAD

**Issue by jimtryon** [freeCodeCamp/freeCodeCamp#52790]
### Is your feature request related to a problem? Please describe.

It's frustrating to have to scroll to the bottom of the page te able to see how much progress you have made when you are completing the challenges. 

### Describe the solution you'd like

I would like for either the information to be arranged in a way that all the critical parts are still easy to see or a different layout for screens that are 15" and less. 

### Describe alternatives you've considered

I haven't come up with any other alternatives other than using PGDN or the Arrow Keys when in the edit field to be able to see the progress bar.

**Comment by jeremylt** [comment]
I don't think we should displace any critical functionality (navbar, instructions, editable region, or submission button) to make room for the progress bar higher on the page. I don't see the progress bar as required to interact with the lesson content, and thus should have lower priority for placement when compared to those 4 critical elements I listed above.

**Comment by moT01** [comment]
To clarify the issue a little bit, you are talking about when you click the "Check Your Code" button and the test pass on [this challenge](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-form-validation-by-building-a-calorie-counter/step-1) for example, the progress bar can be off screen (below) so you can't see it.

Assuming that's the issue - it's not always off the screen, it depends on where you are scrolled to in the editor (pretty sure). We could potentially scroll the editor so the lower jaw is at the bottom of the screen after the tests pass.

**Comment by jeremylt** [comment]
I think someone mentioned that auto scrolling may be an accessibility concern?

**Comment by bbsmooth** [comment]
I think the main accessibility issue with auto scrolling in this scenario is that it could potentially move the keyboard focus indicator out of view, which is a no-no. For example, if the keyboard focus is in the editor and they use the `Ctrl+Enter` shortcut to check their code, then auto scrolling the editor down far enough to show the progress indicator might move the focus indicator out of sight, especially if they have zoomed in considerably.
